resources/i18n/it_IT/cura.po

```diff
@@ -5099,11 +4644,11 @@ msgstr "Unità"
 
 msgctxt "@item:inlistbox"
 msgid "Universal Cura Project"
-msgstr ""
+msgstr "Progetto universale Cura"
 
 msgctxt "@action:description Don't translate 'Universal Cura Project'"
 msgid "Universal Cura Project files can be printed on different 3D printers while retaining positional data and selected settings. When exported, all models present on the build plate will be included along with their current position, orientation, and scale. You can also select which per-extruder or per-model settings should be included to ensure proper printing."
-msgstr ""
+msgstr "I file del progetto universale Cura possono essere riprodotti su diverse stampanti 3D, mantenendo i dati di posizione e le impostazioni selezionate. Quando vengono esportati, tutti i modelli presenti sulla piastra di costruzione saranno inclusi con la posizione, l'orientamento e la scala attuali. È inoltre possibile selezionare che le impostazioni per estrusore o per modello siano incluse per garantire una stampa corretta."
```

Comment:
...don't translate UCP..but honestly when translated is far better
"piastra di costruzione" -> "piano di stampa" same note as before

Reply:
I must say I had the same feeling when reviewing the French translation, but we made the decision to keep Universal Cura Project as like a trademark. I will challenge this decision again, because all languages seem to have it translated so I would have to fix them if we want to be consistent.